### ISSUE TITLE
Check pip-tools installation path

### ISSUE THREAD

**Issue by DocMoebiuz** [eclipse-velocitas/devcontainer-base-images#6]
Error message during devContainer provisioning (e.g. vehicle app python template):
```
FileNotFoundError: [Errno 2] No such file or directory: '/usr/bin/pip'
Call stack:
  File "/usr/local/bin/pip3", line 8, in <module>
    sys.exit(main())
```

This error shows as part of the pip-tools installation. the script assumes that there is `/usr/bin/pip` available, but it should be `/usr/local/bin/pip` - As workaround, we created a symlink as part of the postCreateCommands.sh. 

The actual reason might be either a version compatibility problem or how we are installing pip in the first place (as part of the base image), see
`https://github.com/eclipse-velocitas/devcontainer-base-images/blob/main/Dockerfiles/Python/scripts/install-python.sh`

**Comment by MP91** [comment]
Please verify if this is still valid after merging #3. Might already been fixed.

**Comment by DocMoebiuz** [comment]
fixed by #3
